src/store/action.ts

```diff
@@ -0,0 +1,6 @@
+import {createAction} from '@reduxjs/toolkit';
+
+export const cityСhange = createAction('city change', (value: string) => ({
```

Comment:
экшны именуются обычно без пробелов, посмотри как в демо проекте
```suggestion
export const cityСhange = createAction('ui/changeCity', (value: string) => ({
```